src/graph/validator/test/FetchEdgesTest.cpp

```diff
@@ -21,10 +21,10 @@ class FetchEdgesValidatorTest : public ValidatorTestBase {
 };
 
 TEST_F(FetchEdgesValidatorTest, FetchEdgesProp) {
-  auto src = VariablePropertyExpression::make(pool_.get(), "_VAR1_", kSrc);
+  auto src = ColumnExpression::make(pool_.get(), 0);
```

Comment:
Why change to ColumnExpression?

Reply:
Because the column names of the first three columns in the dataset are fixed, we can use columnExpresion instead of VariableProperyExpression